docs/release_notes.rst

```diff
@@ -20,6 +20,7 @@ Features
 
 Bugs
 ^^^^
+* Make the release graph easier to read when there are many datapoints.
```

Comment:
Can you add a reference to the bug number here, similar to the line below this? Basically, this will do:

```
(`#1172 <https://github.com/fedora-infra/bodhi/issues/1172>`_).
```